index.js

```diff
@@ -63,14 +65,12 @@ module.exports.concat = function (opts, cb) {
     })
     res.on('end', function () {
       var data = Buffer.concat(chunks)
-      if (opts.json) {
-        try {
-          data = JSON.parse(data.toString())
-        } catch (err) {
-          return cb(err, res, data)
-        }
+      if (!opts.json) return cb(null, res, data)
+      try {
+        cb(null, res, JSON.parse(data.toString()))
```

Comment:
It's a very bad idea to call a callback function from within a try block. This means that if the user's function throws an exception than `simple-get` will catch and handle it.

Can you revert this part of this PR?